cli/src/commands/debug/init_local.rs

```diff
@@ -20,15 +20,19 @@ use tracing::instrument;
 
 use crate::cli_util::CommandHelper;
 use crate::command_error::cli_error;
-use crate::command_error::user_error_with_hint;
 use crate::command_error::user_error_with_message;
 use crate::command_error::CommandError;
 use crate::ui::Ui;
 
-/// Create a new repo in the given directory
+/// Create a new repo in the given directory using the proof-of-concept native
+/// backend
 ///
-/// If the given directory does not exist, it will be created. If no directory
-/// is given, the current directory is used.
+/// The proof-of-concept backend is called "local" because it does not support
+/// cloning, pushing, or fetching repositories or commits.
```

Comment:
I have no idea if this is true.

Reply:
The alternative would be to say

> This backend does not currently support
> cloning, pushing, or fetching repositories or commits.

I'm fine with either.

Reply:
I'll just go with this, we can change it later.